lib/bundlex/os_deps.ex

```diff
@@ -0,0 +1,204 @@
+defmodule Bundlex.OSDeps do
```

Comment:
I'm thinking about moving this to `Budlex.Toolchain.Common.Unix.OsDeps`, as it's quite unix-specific, though the name would get scaringly java-style :P